packages/live-share-media/src/MediaPlayerSynchronizer.ts

```diff
@@ -149,7 +154,11 @@ export class MediaPlayerSynchronizer extends EventEmitter {
                                 this._logger.sendTelemetryEvent(TelemetryEvents.MediaPlayerSynchronizer.SeekingPlayerToStartPosition, null, {position: details.seekTime})
                                 this._player.currentTime = details.seekTime!;
                             }
-                            this._player.play();
+                            await this._player.play()
+                                .catch((error: any) => {
+                                    // Error can occur when user has not sufficiently interacted with browser tab
+                                    playerError = error;
+                                });
                         }
```

Comment:
You should use a try{ }catch() to properly catch the playError

Reply:
Just rename "playError" to "error" and add a try/catch around the entire switch statement.